elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -478,6 +525,20 @@ def grade(element_html, data):
         num_initial_correct, true_answer_length = grade_dag(submission, depends_graph, group_belonging)
         first_wrong = -1 if num_initial_correct == len(submission) else num_initial_correct
 
+        if feedback_type == 'first-wrong':
+            for i, block in enumerate(student_answer):
+                if i >= num_initial_correct: break
```

Comment:
minor: use a list slice here to control the for loop instead of a break, just like you did a few lines below this.

Reply:
```
for block in student_answer[:first_wrong + 1]:
```

Reply:
oooh I forgot I can just directly suggest an edit. let me do that.